### DISCUSSION TITLE
OudieN and  xcvario Bluetooth

### DISCUSSION THREAD

**Discussion by nferruzzi** [iltis42/XCVario#179] [Q&A] [answered]
Has anyone been able to connect xcvario and oudie (or any other android device) via Bluetooth ? I can't get xcvario discovered by any of the android / iOS devices I've tried. It usually doesn't show up at all, just once on s device it did show up but then failed to connect. Latest firmware updated and Wi-Fi working fine . Thanks in advance

**Comment by iltis42** [comment]
Maybe there is a hardware defect, there is no such report until now from the field and majority of XCVario users are using Bluetooth with success on android based devices, also iOS gadgets should work (even there is limited use as of absence of XCSoar and LK8000).  Propose once you don't need the device for a while sent it for service to do a check.

**Comment by iltis42** [comment, marked as answer]
There is a report from a user who is using Oudie bluetooth connection with the XCVario, and here is how he did.
Note the Oudie is using an old Bluetooth standard (2.0 maybe), that still requires a password to be entered.
The procedure is here below:

To connect an Oudie to an XC Vario:

- Config XC Vario wireless as Bluetooth.
- Config XC Vario NMEA protocol as Cambridge. 

Then on the Oudie...

- Select the SeeYou Mobile Settings / Hardware tab. Configure Com5 Bluetooth:  38400, none, 8, 1, DTR and RTS
- In SeeYou Mobile Settings / User Interface, select Bluetooth.
- Search for your XC Vario.
- Connect. The PIN is 1234. (the PIN is not listed in the vario setup screen on this version of XC Vario, but it may be there now)
- In SeeYou Mobile Settings / Hardware, click the Terminal button, then Yes.
    You should see this....

    Success!

**Reply by mspudi** [reply]
Add agreement LK8000
